translations/ai_heroes.tex/pl.po

```diff
@@ -106,20 +119,36 @@ msgid ""
 "Magic Card \\includesvg[height=10px]{\\svgs/magic.svg} is drawn.  If an AI "
 "Hero is instructed to draw a card, they will draw and resolve "
 "\\textbf{another card} from the AI Deck."
-msgstr ""
+msgstr "Bohaterowie AI używają dwóch Talii Automatycznych podczas Walki:"
+"\\textbf{Talii AI} oraz \\textbf{Talii Zaklęć}. Talia AI jest budowana z wykorzystaniem"
+"trzech rodzajów kart AI: Siły \\includesvg[height=10px]{\\svgs/might.svg},"
+"Magii \\includesvg[height=10px]{svgs/magic.svg} oraz Umiejętności \\includesvg[height=10px]{svgs/skill.svg}."
+"Każdy Scenariusz Kampanii podaje liczbę każdego rodzaju (typu) Karty AI wchodzącej w skład Talii AI."
+"Karty Siły \\includesvg[height=10px]{svgs/might.svg} oraz Umiejętności \\includesvg[height=10px]{svgs/skill.svg}"
+"mają dwie wersje. Wybierz te karty \\textbf{losowo} w trakcie budowania Talii."
+"Jeśli w Talii są karty Umiejętności, wyszukaj i wybierz je odkładając na bok."
+"Zbuduj \\textbf{Talię Zaklęć AI} oddzielając wskazane Zaklęcia od Zwykłej (regularnej) Talii Zaklęć."
+"Potasuj Talie AI i Talię Zaklęć podczas przygotowania po ich utworzeniu."
```

Comment:
```suggestion
"Po ich utworzeniu potasuj Talie AI i Talię Zaklęć podczas przygotowania gry."
```

Reply:
Chyba nie zauwazylem swojego bledu - i teraz sie zastanawiam czy obie wersje slowa Talia (liczba pojedyncza i mnoga) obok siebie powinny byc w jednej "liczbie", tzn albo mnoga albo pojedyncza...

Reply:
Racja, poprawiłem.

Reply:
A talia zaklęć to ma być talia Magii...